sound/soc/sof/loader.c

```diff
@@ -192,6 +192,31 @@ static int ext_man_get_dbg_abi_info(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+static int ext_man_get_config_data(struct snd_sof_dev *sdev,
+				    const struct sof_ext_man_elem_header *hdr)
+{
+	const struct sof_ext_man_config_data *config_data =
+		container_of(hdr, struct sof_ext_man_config_data, hdr);
+	int i;
+
+	/* TODO: Currently CAVS_LPRO only supported */
+	for (i = 0; i < SOF_EXT_MAN_CONFIG_LAST_ELEM; i++)
+		switch (config_data->elems[i].token) {
+		case SOF_EXT_MAN_CONFIG_CAVS_LPRO:
+			sdev->clk_config = config_data->elems[i].value;
+			dev_dbg(sdev->dev, "CAVS_LPRO: clk_config is %d\n", sdev->clk_config);
```

Comment:
can we improve this debug message further like this? 
dev_dbg(sdev->dev, "FW clock config: %s\n", sdev->clk_config ? "LPRO" : "HPRO");